cli/src/config/templates.toml

```diff
@@ -66,6 +66,8 @@ separate(" ",
 )
 '''
 
+duplicate_description = ''
```

Comment:
Perhaps, the default should be `'description'`. The behavior shouldn't change depending on the optimization for the default template.

Can you also move this after `draft_commit_description`? Appears that the templates are sorted lexicographically.

Reply:
> Perhaps, the default should be `'description'`. The behavior shouldn't change depending on the optimization for the default template.

Not sure what you mean -- the exact behavior wouldn't change at all due to the `is_empty` check. We're just changing the optimized case.

Also, to clarify: no matter if the template is `'description'` (optimized out) or `' description '`, the resulting commits will be identical.

Reply:
> Not sure what you mean -- the exact behavior wouldn't change at all due to the `is_empty` check.

I mean the behavior shouldn't change if we removed the `if`. It should just exist for optimization purpose. Since an empty string is a valid template expression, unfortunately, it should be evaluated to `""`, not to `description`.

FWIW, if user defined `description` alias, the result would go awry. It might be safer to just always render the template.

Reply:
Good point, thanks!